frameworks/hdfs/integration/tests/test_upgrade.py

```diff
@@ -26,32 +34,36 @@ def setup_module(module):
 
 @pytest.mark.upgrade
```

Comment:
Should this be marked `sanity`+`skip` like hello-world's version?: https://github.com/mesosphere/dcos-commons/blob/master/frameworks/helloworld/integration/tests/test_upgrade.py#L31